application/src/main/java/com/sanction/thunder/authentication/ThunderAuthenticator.java

```diff
@@ -1,24 +1,28 @@
 package com.sanction.thunder.authentication;
 
-import io.dropwizard.auth.AuthenticationException;
 import io.dropwizard.auth.Authenticator;
 import io.dropwizard.auth.basic.BasicCredentials;
 
 import java.util.List;
+import java.util.Objects;
 import java.util.Optional;
 
 import javax.inject.Inject;
 
+/**
+ * Implements the Dropwizard {@code Authenticator} and can be registered with Jersey
```

Comment:
So with something like this I think we can inherit JavaDocs from the super class then add our own summary on top. I'll show an example in my writeup.

Reply:
I like that suggestion. I'm not super familiar with Javadoc so an example would help a lot